Catppuccin VS Themes/Catppuccin Latte.vstheme

```diff
@@ -144,7 +144,7 @@
         <Foreground Type="CT_RAW" Source="FF4C4F69" />
       </Color>
       <Color Name="ListItemMouseOver">
-        <Background Type="CT_RAW" Source="FFACB0BE" />
```

Comment:
Not sure about the bg for this.. goes from blue to bright yellow

Reply:
I'm not sure you commented on the correct line? The color shift is from darker to brighter gray/white because the first HEX value is the opacity value. 

![image](https://github.com/catppuccin/visual-studio/assets/4233281/6950274b-5c65-4d5a-98c9-a68ce4445a8